ptbr/training_cli.py

```
@@ -0,0 +1,1193 @@
+    python -m ptbr.training_cli --output-folder ./runs config.yaml
+    python -m ptbr.training_cli --output-folder ./runs --resume config.yaml
+"""
+
+from __future__ import annotations
+
+import copy
+import logging
+import os
+import sys
+import time
+import copy
```

Ruff (0.15.1)

[error] Redefinition of unused `copy` from line 16: `copy` redefined here

Remove definition: `copy`

(F811)